### ISSUE TITLE
sound effects are missing!!

### ISSUE THREAD

**Issue by patcami91** [nikocall/IMTAssignment2016#4]
Arent all the pickups suppose to have sound effects once you collect them!!

**Comment by nikocall** [comment]
Yes some of  the sound effects are missing, this is because the the sound effects have not been linked yet. They will be linked as soon as they according sounds have been chosen.
